test/integration-test-main.sh

```diff
@@ -568,12 +568,13 @@ function test_concurrent_writes {
 
 function test_open_second_fd {
     describe "read from an open fd"
-    rm -f ${TEST_TEXT_FILE}
-    RESULT=$( (echo foo ; wc -c < ${TEST_TEXT_FILE} >&2) 2>& 1>${TEST_TEXT_FILE})
+    rm_test_file secound_fd_file
```

Comment:
typo: second

Reply:
I'm sorry for this typo, I will merge fixed this soon.